pallets/liquid-staking-two/src/lib.rs

```diff
@@ -247,45 +249,116 @@ pub mod pallet {
             origin: OriginFor<T>,
             era_index: EraIndex,
         ) -> DispatchResultWithPostInfo {
+            ensure_signed(origin)?; //TODO(Alan): Who can call this extrinsic?
```

Comment:
We can refer to this:
https://github.com/parallel-finance/parallel/blob/master/pallets/liquid-staking/src/lib.rs#L347